### PR TITLE
Initial documentation for Hotel Service 4

### PR DESCRIPTION
**Pull request by kunalshah01**
Please help review and make it available as preview next version of Hotel Service API (v4) for hotel suppliers

### CODE REVIEW

File: src/_data/sidebars/api-reference.yml [hunk 1]
```diff
@@ -122,6 +122,8 @@
         url: /api-reference/direct-connects/hotel-service-2/Appendix.html
       - title: XSD-schema
         url: /api-reference/direct-connects/hotel-service-2/xsd-schema.html
+- title: Direct Connect - Hotel v4
```
[the remark this answers is not in the document]
Comment (kunalshah01) [reply src/_data/sidebars/api-reference.yml]: Removed

File: src/api-reference/direct-connects/hotel-service-4/yaml/HotelService4.yaml [hunk 2]
```diff
@@ -0,0 +1,1576 @@
+openapi: 3.0.0
```
[the remark this answers is not in the document]
Comment (kunalshah01) [reply src/api-reference/direct-connects/hotel-service-4/yaml/HotelService4.yaml]: Changed to JSON instead

File: src/api-reference/direct-connects/hotel-service-4/HotelService4.markdown [hunk 3]
```diff
@@ -0,0 +1,4565 @@
...
+  ],
+  "numGuests": 1,
+  "searchSessionToken": "b41168ba-7ee1-4b68-9934-47f5c55337d6"
+}
+
+```
+
+### Properties
+
+|Name|Type|Required|Restrictions|Description|
```
[the remark this answers is not in the document]
Comment (kunalshah01) [reply src/api-reference/direct-connects/hotel-service-4/HotelService4.markdown:1636]: I checked all response properties and they are already part of Schema section. We only have HTTP codes and pointers to schema with each endpoint

**Comment by kunalshah01** [comment]
@jonikamelcher sorry about long delay on this. But we later realized that we needed to wait for some more changes before it becomes available as preview. All anticipated changes are in now. If you can please review and help with the merge process. Our goal is to make this available for our "preview" suppliers soon.

**Review by jonikamelcher — APPROVED** [review]
:shipit:

**Pull request merged**